data/distrodefs/fedora/imagetypes.yaml

```diff
@@ -532,6 +532,91 @@
       aarch64:
         <<: *iot_simplified_installer_partition_tables_x86
 
+  supported_options_lists:
+    # common options supported by all disk image types this includes everything
+    # that is not specific to installers or ostree-based images
+    supported_options_disk: &supported_options_disk
+      - "name"
```

Comment:
(wondering) Should we make name/version/description implicit? They are not really concepts that are relevant for the image types, i.e. it would be weird to have an image type that refuses a blueprint with a "name"?

Similar wondering about "distro", it goes away in UBP and its a bit of a odd one, so maybe the image types should not list it?

Reply:
I think that hiding it makes it slightly more difficult to get up to speed with it. It's supposed to be a list of blueprint fields, just make it 1:1 without any magic.